code/modules/paperwork/stamps.dm

```diff
@@ -12,6 +12,68 @@
 	item_color = "cargo"
 	pressure_resistance = 2
 	attack_verb = list("stamped")
+	/// used for on human stamp marks, each mark will be colored this before adding to image in human stamp_marks
+	var/stamp_color = "#6ca151"
+	/// this is defined uniquely for each subtype, used in examine.dm for when humans are stamped.
+	var/stamp_description = "..."
+
+/obj/item/stamp/afterattack(atom/target, mob/user, proximity, params)
+	. = ..()
+	if(!ishuman(target) || !ishuman(user) || !proximity)
+		return
+
+	var/mob/living/carbon/human/H = target
+
+	if(length(H.stamp_marks.overlays) == 50)	//no more than 50 marks on a person, otherwise infinite laggy stamp marks
```

Comment:
```suggestion
	if(length(H.stamp_marks.overlays) >= 50)	//no more than 50 marks on a person, otherwise infinite laggy stamp marks
```
Dont want someone somehow getting 51 then applying infinite amounts. Also 50 still seems way too high (Keep in mind a lot of the people playing this game dont have hyper-powerful computers, and 50 overlays is technically processing an extra 51,200 pixels.

Reply:
I dropped it down to 20, which ought to cut that number in half at least. I have made the suggested changes to insure that if somehow more are added the code addresses that.